app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RegistrationsController < Devise::RegistrationsController
+  protected
+
+  def after_sign_up_path_for(resource)
```

Comment:
You're passing `resource` as an argument, but not using it in the method body. Do you plan on using it going forward?

Reply:
I was following a tutorial and was not sure what the resource argument was for.  It didn't break anything, but I will do some research on it to verify it's purpose.  Seems to me that you are saying that I don't need it there if it is unused?

Reply:
It look like I have two options (resource or scope). If the controller does not find a resource or hook then it will default to the root path.  If I use scope then I can define the url path.  I changed it to scope, which I believe is correct.  I will do a new commit and ping you on slack.